src/WORKSPACE

```diff
@@ -34,6 +34,7 @@ git_repository(
     name = "com_github_nelhage_rules_boost",
     commit = "9f9fb8b2f0213989247c9d5c0e814a8451d18d7f",
     remote = "https://github.com/nelhage/rules_boost",
+    shallow_since = "1570056263 -0700",
```

Comment:
not sure why this is included in this PR :thinking:

Reply:
Bazel was giving warnings it was needed to produce "fully canonical reproducible" dependencies so I added it